crates/rust-project-goals-cli/src/cfp.rs

```diff
@@ -0,0 +1,376 @@
+                    } else {
+                        println!("Would skip updating SUMMARY.md (already contains a non-placeholder entry for {})", timeframe);
+                    }
+                    return Ok(());
+                }
+            }
+        }
+    }
+    
+    // If we get here, the section doesn't exist, so we need to add it
+    let new_section = format!("\n{}", new_section_content);
+    
+    // Find a good place to insert the new section
+    // Look for the last timeframe section or insert at the beginning
+    // Match both lowercase and uppercase H
+    let re = Regex::new(r"# ⏳ \d{4}[hH][12] goal process").unwrap();
+    let mut new_content = content.clone();
+    
+    if let Some(last_match) = re.find_iter(&content).last() {
+        // Find the end of this section (next section or end of file)
+        if let Some(next_section_pos) = content[last_match.end()..].find("\n# ") {
+            let insert_pos = last_match.end() + next_section_pos;
+            new_content.insert_str(insert_pos, &new_section);
+        } else {
+            // No next section, append to the end
+            new_content.push_str(&new_section);
+        }
+    } else {
+        // No existing timeframe sections, insert at the beginning
+        new_content = new_section + &content;
+    }
+    
+    // Write the updated content back to SUMMARY.md
+    if !dry_run {
+        File::create(summary_path)
+            .with_context(|| format!("Failed to open SUMMARY.md for writing"))?
+            .write_all(new_content.as_bytes())
+            .with_context(|| format!("Failed to write to SUMMARY.md"))?;
+        
+        println!("Updated SUMMARY.md with {} section", timeframe);
+    } else {
+        println!("Would update SUMMARY.md with {} section", timeframe);
+    }
+    
+    Ok(())
+}
+
+/// Updates the src/README.md with information about the new timeframe
+fn update_main_readme(timeframe: &str, lowercase_timeframe: &str, dry_run: bool) -> Result<()> {
+    let readme_path = "src/README.md";
+    let mut content = String::new();
+    File::open(readme_path)
+        .with_context(|| format!("Failed to open README.md"))?
+        .read_to_string(&mut content)
+        .with_context(|| format!("Failed to read README.md"))?;
+    
+    // Extract year and half from timeframe
+    let _year = &timeframe[0..4];
+    let half = &timeframe[4..].to_lowercase();
+    
+    // Determine the months based on the half
+    let (start_month, end_month) = if half == "h1" {
+        ("January", "June")
+    } else {
+        ("July", "December")
+    };
+    
+    // Create the new section to add with capitalized H
+    let capitalized_timeframe = format!("{}H{}", &timeframe[0..4], &timeframe[5..]);
+    let new_section = format!(
+        "\n## Next goal period ({})\n\n\
+         The next goal period will be {}, running from {} 1 to {} 30. \
```

Comment:
```suggestion
         The next goal period will be {}, running from the start of {} to the end of {}. \
```

It's probably not always a 30 day month...?